Add Navigation theme

lcgjapan opened the pull request:
# Navigation Theme

## Description
Modern minimalist ring navigation theme with dynamic ripple animations and dual background system.

## Features
- ✨ Dynamic Ring Ripple Animations
- 🎭 Natural Sway Motion for icon expansion  
- 🌈 Dual Background Layer System
- 🎯 Performance Optimized (60fps)
- 💫 Multi-layer Ring Glow Effects
- 🎨 Customizable dual-screen background colors

## Authors
Rena and LCG

## License
CC0-1.0

## Preview
![Navigation](themes/Navigation/preview.jpg)

yar2000T reviewed (changes requested):
Hello @lcgjapan ! Your theme is awesome, but could you please add it to the ``README.md ``as follows:

```
| [**Navigation**](./themes/nord/) <br>Modern minimalist ring navigation theme with dynamic ripple animations and dual background system. <br> <br> [![Download](https://img.shields.io/badge/🡇-Download-blue?style=flat)](https://github.com/kando-menu/menu-themes/releases/latest/download/Navigation.zip)                                                                                                                                                                                                               | ![preview](./themes/Navigation/preview.jpg)                 |
```

lcgjapan commented:
> 你好@lcgjapan！您的主题很棒，但您能否将其添加到`README.md `以下内容：
> 
> ```
> | [**Navigation**](./themes/nord/) <br>Modern minimalist ring navigation theme with dynamic ripple animations and dual background system. <br> <br> [![Download](https://img.shields.io/badge/🡇-Download-blue?style=flat)](https://github.com/kando-menu/menu-themes/releases/latest/download/Navigation.zip)                                                                                                                                                                                                               | ![preview](./themes/Navigation/preview.jpg)                 |
> ```
Hi @yar2000T! 👋

Thank you so much for reviewing my Navigation theme PR! 😊
I've just updated the README.md file as you suggested - added Navigation theme to the themes list.

Really appreciate your quick feedback and guidance! 🙏
The theme is now complete with all required files and documentation.

Thanks again for helping make this contribution better! ✨

yar2000T commented:
Thank you! But I forgot to say, could you please make your folder name lowercase? And if you will do this, replace old text with this: ``| [**Navigation**](./themes/navigation/) <br>Modern minimalist ring navigation theme with dynamic ripple animations and dual background system. <br> <br> [![Download](https://img.shields.io/badge/🡇-Download-blue?style=flat)](https://github.com/kando-menu/menu-themes/releases/latest/download/navigation.zip)                                                                                                                                                                                                               | ![preview](./themes/navigation/preview.jpg)                 |``

lcgjapan commented:
> 谢谢！不过，我忘了说，请把文件夹名称改成小写。如果可以的话，请用下面的代码替换旧文本：
> 
> ```
> | [**Navigation**](./themes/navigation/) <br>Modern minimalist ring navigation theme with dynamic ripple animations and dual background system. <br> <br> [![Download](https://img.shields.io/badge/🡇-Download-blue?style=flat)](https://github.com/kando-menu/menu-themes/releases/latest/download/navigation.zip)                                                                                                                                                                                                               | ![preview](./themes/navigation/preview.jpg)                 |
> ```

Hi @yar2000T! 

I've completed the requested changes:
✅ Renamed Navigation folder to navigation (lowercase)
✅ Updated README.md with the exact code you provided
✅ All file paths now use lowercase: ./themes/navigation/

Thank you for the guidance! Please review when convenient. 😊

Schneegans commented:
Hey, this looks really great! The only thing I am a bit hesitant about is that the center text configuration is not compatible with the latest changes with the circle wrapping and the automatic font size reduction.

If we do not want to break this theme with the next update, it would be cool if you could make it compatible right away. You can download a version of Kando which includes the latest changes to the center text from [here](https://github.com/kando-menu/kando/actions/runs/16547833259).

Your Neo-Ring theme works, the neon-lights-color does not work either. I suspect that it is caused by hard-coding many values in the `.center-text` style:

```css
  max-width: 160px !important;
  min-width: 120px !important;
  max-height: 60px !important;
  
  word-wrap: break-word !important;
  word-break: break-all !important;
  white-space: normal !important;
  text-align: center !important;
  overflow: hidden !important;
  text-overflow: ellipsis !important;
  
  display: flex !important;
  align-items: center !important;
  justify-content: center !important;
  padding: 6px 8px !important;
```

Maybe you could remove most of them and see what is really necessary...

lcgjapan commented:
> And yes, it would be awesome, if you could fix the neon-lights-color theme as well!

Hi @Schneegans!

Thank you for the feedback! ✅ I've now updated BOTH themes for Kando 2.0 compatibility:

**Navigation Theme:**
- Removed all hard-coded !important declarations from .center-text
- Set centerTextWrapWidth to 140px for perfect fit within inner circle
- Preserved signature glow effects

**Neon-lights-color Theme:**
- Applied the same optimization approach
- Set centerTextWrapWidth to 100px (smaller due to neon ring design)
- Preserved signature neon glow effects

**Results:**
- Both themes now display center text perfectly with automatic scaling
- Long text gets intelligently wrapped and resized
- Text areas perfectly match circular boundaries
- Full compatibility with Kando 2.0's circle wrapping and font size reduction

Both themes maintain their unique visual identities while being fully compatible with the new text system. Ready for final review! 😊
